src/resmom_win/start_exec.c

```diff
@@ -1083,12 +1086,24 @@ set_homedir_to_local_default(job *pjob, char *username)
 			return ("");
 		}
 
-		strcpy(lpath, default_local_homedir(username,
-			pp->pw_userlogin, 0));
-		return (lpath);
+		if(default_local_homedir(username,
+			pp->pw_userlogin, 0) != NULL) {
+			strcpy(lpath, default_local_homedir(username,
```

Comment:
Why calling this twice ? Call once only and save in a variable and do strcpy on that